caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1534,37 +1552,71 @@
      * @private
      */
     this._setDefaultUnitTypes = function (_viewer) {
-      var unitTypes = this._getActiveForm()
-        .getConfiguration()
-        .getDefaultUnitTypes()
 
-      for (var columnName in unitTypes) {
+      //
+      var allColumnOptions = this._getActiveForm().getConfiguration().getColumnOptions()
+      var allUnitTypes = {}
+      for (var columnOption in allColumnOptions) {
+        var column = allColumnOptions[columnOption]
+        if (column.header)
+        {
```

Comment:
Sorry to be picky.  For consistency, please put if/else braces inline.

Reply:
Fixed